src/test/scala/nl/knaw/dans/easy/schema/AgreementsSpec.scala

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) 2012 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.schema
+
+import scala.util.Success
+
+class AgreementsSpec extends TestSupportFixture {
+
+  override val schemaDir: String = lastXSD("bag/metadata/agreements")
+
+  "validator" should "succeed" in {
+    val xml =
+      <agreements xsi:schemaLocation="http://easy.dans.knaw.nl/schemas/bag/metadata/agreements/ https://easy.dans.knaw.nl/schemas/bag/metadata/agreements/2018/12/agreements.xsd"
```

Comment:
I would much rather see an example in `/docs/examples` and validate that to be true, just like you did in `DdmSpec`.
That is, if my concerns in the previous comment were not valid.

Reply:
@lindareijnhoudt 
Were would you want this example. Most are in the docs directory, but there is also a https://github.com/DANS-KNAW/easy-schema/blob/2ed23fb1d/src/main/assembly/dist/collections/example-dmo-collection.xml